packages/rmf-dashboard-framework/vitest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import { vi } from 'vitest';
+
+/* @ts-ignore */
```

Comment:
We should use `ts-expect-error` instead, with a comment explaining why that is the case.